plugins/wdc/wdc-nvme.c

```diff
@@ -2060,6 +2071,46 @@ static int wdc_do_cap_telemetry_log(int fd, char *file, __u32 bs, int type, int
 	case 3:
 		full_size = (le16_to_cpu(hdr->dalb3) * WDC_TELEMETRY_BLOCK_SIZE) + WDC_TELEMETRY_HEADER_LENGTH;
 		break;
+	case 4:
+		err = nvme_identify_ctrl(fd, &ctrl);
+		if (err) {
+			perror("identify-ctrl");
+			goto close_output;
+		}
+		
+		if (posix_memalign(&buf, getpagesize(), nvme_wdc_feat_buf_len[NVME_FEAT_HOST_BEHAVIOR])) {
+			fprintf(stderr, "can not allocate feature payload\n");
+			errno = ENOMEM;
+			err = -1;
+			goto close_output;
+		}
+		memset(buf, 0, nvme_wdc_feat_buf_len[NVME_FEAT_HOST_BEHAVIOR]);
+
+		err = nvme_get_feature(fd, NVME_NSID_ALL, NVME_FEAT_HOST_BEHAVIOR, 0, 0,
+				0, nvme_wdc_feat_buf_len[NVME_FEAT_HOST_BEHAVIOR], buf, &result);
+		if (err > 0) {
+			nvme_show_status(err);
+		} else if (err < 0) {
+			perror("get-feature");
+		} else {
+			if ((ctrl.lpa & 64)) {
```

Comment:
Does it make more sense to use 0x40 instead of 64 here and above in the base code?   To me, it seems more obvious which bit you're checking with 0x40.